index.js

```diff
@@ -37,9 +37,13 @@ if (argv.help || argv.h) {
   console.log('  -h, --help   show this help')
   console.log('  -q, --quiet  don\'t output stdout from tests unless an error occors')
   console.log('  --verbose    output a lot of information while running')
+  console.log('  -n           do a dry-run (don\'t actually execute anything)')
```

Comment:
Why `-n`? I haven't seen that used as a dry-run option before... what about something more explicit, like `--dry-run`?

Reply:
I'm happy either way.

I've noticed `-n` used in *some* tools for dry-run:
```
$ man make
...
       -n, --just-print, --dry-run, --recon
            Print the commands that would be executed, but do not execute them.

$ man rsync
...
       -n, --dry-run               show what would have been transferred

$ git help clean
...
       -n, --dry-run
           Don’t actually remove anything, just show what would be done.
```

However, it certainly isn't universal. E.g.

```
$ git help commit
       --dry-run
           Do not create a commit, but show a list of paths that are to be committed, paths with
           local changes that will be left uncommitted and paths that are untracked.
```

I'll change it to `--dry-run`.

Reply:
commit f4049f9